File: Piko/Sources/SFOutput.m
//
//  SFOutput.m
//
//  Copyright (c) 2014 Satoshi Muraki. All rights reserved.
//

#import "SFOutput.h"

@implementation SFOutput

- (instancetype)initWithName:(NSString *)name
{
    self = [super init];
    if (self != nil) {
        _name = [name copy];
    }
    return self;
}

@end

File: Piko/Sources/SFOutput.swift
//
//  SFOutput.swift
//
//  Copyright (c) 2014 Satoshi Muraki. All rights reserved.
//

import Foundation

@objc class SFOutput: NSObject {
    let name: String
    weak var owner: SFNode?
    weak var destination: SFInput?

    init(name: String) {
        self.name = name
    }
}
